webviz_subsurface/_utils/colors.py

```diff
@@ -3,7 +3,7 @@
 from typing import Tuple
 
 
-class StandardColors(Enum):
+class StandardColors(str, Enum):
```

Comment:
Should we here instead now use https://github.com/equinor/webviz-subsurface/blob/7c35594495cebb4d57584e7c2f8b347751b9f778/webviz_subsurface/plugins/_group_tree/_types.py#L1?

Reply:
Thanks, didn't know it existed. Will test :)

Reply:
👍 It is quite new. The idea is that it then should be hopefully easier to deprecate + start using std.lib https://docs.python.org/3/library/enum.html#enum.StrEnum when `python lower bound >= 3.11`